resources/lang/id/layout.php

```diff
@@ -0,0 +1,263 @@
+<?php
+
+/**
+ *    Copyright 2015-2017 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+return [
+    'defaults' => [
+        'page_description' => 'osu! - Rhythm is just a *click* away! Dengan Ouendan/EBA, Taiko dan mode permainan original, serta level editor yang berfungsi.',
```

Comment:
ini "Rhythm is just a *click* away!" apa disengajakan pakai bahasa Inggris? kalau tidak coba gunakan "Ritme hanya dalam sebuah *klik*!"

Reply:
Saya rasa itu slogan-nya osu!, makanya saya keep. 👍